### PR TITLE
Updates for Serilog 4

### PR DESCRIPTION
**Pull request by Numpsy**
Just code changes so far - haven't changed the build to use snupkg for symbols

Update: changed to publish a snupkg now

### CODE REVIEW

**Comment by Numpsy** [comment]
Hmm, the version in the CI build nuget package doesn't seem quite right (says 0.0.0.0 for the strong name) -

image

**Comment by nblumhardt** [comment]
:thinking: Hmm... that's not quite right...

**Comment by Numpsy** [comment]
Managed to miss the ```<GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>``` :-(

**Comment by Numpsy** [comment]
Updated to publish a snupkg instead of using embedded symbols

File: src/Serilog.Enrichers.Process/Enrichers/ProcessIdEnricher.cs [hunk 1]
```diff
@@ -45,10 +45,8 @@ private static int GetProcessId()
 #if FEATURE_ENVIRONMENT_PID
             return System.Environment.ProcessId;
 #else
-            using (var process = System.Diagnostics.Process.GetCurrentProcess())
-            {
-                return process.Id;
-            }
+            using var process = System.Diagnostics.Process.GetCurrentProcess();
```
Comment (nblumhardt) [inline src/Serilog.Enrichers.Process/Enrichers/ProcessIdEnricher.cs:48]: All targets now support `Environment.ProcessId`, might be nice to clobber this?

Comment (nblumhardt) [reply src/Serilog.Enrichers.Process/Enrichers/ProcessIdEnricher.cs:48]: Ah sorry, my mistake, the automatic redirect in the .NET docs fooled me :-)

**Pull request merged**